ddspipe_core/include/ddspipe_core/configuration/DdsPipeConfiguration.hpp

```diff
@@ -32,6 +33,15 @@ namespace eprosima {
 namespace ddspipe {
 namespace core {
 
+//! Possible kinds of the endpoint
+ENUMERATION_BUILDER(
+    DiscoveryTrigger,
+    READER,     //! The discovery callbacks get triggered by the discovery of a reader.
```

Comment:
Use lower case to simplify your life (and reviewer's).

Reply:
"Example is not another way to teach, it's the only way." — Albert Einstein

Reply:
Besides, I wouldn't simplify anything. I would have to rename all the names to save a copy, which serves as a good reminder to add `to_uppercase` and `to_lowercase` functions that return the string in uppercase or lowercase.

Reply:
"The fool knows after he's suffered." — Hesiod